tasks/data/vlm_data_process.py

```diff
@@ -135,6 +135,79 @@ def process_sample_qwen2_5_vl(
 
     return [tokenized_example]
 
+def process_sample_qwen3_vl(
```

Comment:
@piyifan123 Please check if there's any conflict with the origin `process_sample_qwen3_vl`. If no, use this new function?